src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/NomisEventOutcomeMapper.kt

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import uk.gov.justice.digital.hmpps.whereabouts.model.AbsentReason
+import java.util.HashMap
+
+internal class NomisEventOutcomeMapper {
+    init {
```

Comment:
should this init block be in the companion object?

Reply:
:nauseated_face: 
No reason in kotlin to use a mutable map anyway to store this data:
```kotlin
  companion object {
    private val eventOutcomes = mapOf(
        AbsentReason.AcceptableAbsence to EventOutcome("ACCAB"),
        AbsentReason.NotRequired to EventOutcome("NREQ"),
        AbsentReason.ApprovedCourse to EventOutcome("ACCAB"),

        // unpaid absences
        AbsentReason.SessionCancelled to EventOutcome("CANC"),
        AbsentReason.RestInCell to EventOutcome("REST"),
        AbsentReason.Sick to EventOutcome("REST"),
        AbsentReason.RestDay to EventOutcome("REST"),

        //unpaid absences that trigger automatic iep warnings
        AbsentReason.Refused to EventOutcome("UNACAB"),
        AbsentReason.UnacceptableAbsence to EventOutcome("UNACAB"))
  }
```